less/index.less

```diff
@@ -57,3 +57,267 @@ html, body {
     height: 100%;
 }
 
+@body-size: 1.2rem;
+@fonts: Noto Sans;
+@fonts-2: Calibri;
+@font-color-main: #9591AF;
+@font-color-second: #91AFA5;
+@font-color-p: rgb(88, 85, 85);
+
```

Comment:
Good use of variables.
Another good using to use variables is that you don't have to remember the color codes and it helps typo's from ruining your code.